### DISCUSSION TITLE
How to improved or quantified the impact of food ?

### DISCUSSION THREAD

**Discussion by Carrie-nothing** [Open-Systems-Pharmacology/Forum#1017] [Questions & Problems]
Hello team,
I wonder how I can make the figure much better?
![image](https://user-images.githubusercontent.com/92715703/141429794-c15ea668-f4e6-44be-8bb7-b5471564b5ca.png)
I have already add a fed event,but that is not enough.
![image](https://user-images.githubusercontent.com/92715703/141429890-38542d79-9397-4ff1-8e27-322abab38a49.png)

It will be great if you could give me some feedback. Thank you very much.

Have a nice day.

Best,
Carrie.

**Comment by StephanSchaller** [comment]
You could try increasing the gastric emptying time. What process is supposedly reducing the rate of absorption for the second dose?

Best,
Stephan

**Reply by Carrie-nothing** [reply]
Thank you so much for your answer,Stephan.

The second dose was affected by food and the literature showed that food reduced the maximum bioavailability of the drug by 50 percent.

So should I use mobi?or just pksim?

Best,
Carrie

**Reply by Carrie-nothing** [reply]
Please allow me to add information.

The first dose is fasting condition and the second dose is feeding condition.
If both doses are fed, I can adjust the solubility under feeding, gi time, etc.
But the problem is that the delivery is complicated, and I don't know how to achieve two different absorption in one simulation?

**Reply by StephanSchaller** [reply]
If you increase caloric content and fraction solid of a meal, the release for stomach will go down even further.
But it may also be related to a different release of the drug in fed conditions, and you can create two separate oral administrations with their own formulation/release profile to account for this (e.g. use Weibull as Formulation and adjust the release profile to be slow).
This way you can control the second dose absorption behavior (to some extent) independently from the first.
Do you know the reasons for the reduced bioavailability (on a mechanistic level)?
